### DISCUSSION TITLE
想买树莓派，大伙有什么建议吗

### DISCUSSION THREAD

**Discussion by ChaoKaiADG** [share121/inter-knot#5561] [公告]
想买两个pi来玩，一个当服务器，一个用来当workstation。

**Comment by 6078HITHEDAY** [comment]
推荐pi5

**Comment by EcoFur** [comment]
推荐RP2040（微控制器）不能做服务器 ~~（其实也可以）~~ ，可以看我的项目（自豪）：https://oshwhub.com/fuck_me/rp2040mini

**Reply by Yizakl** [reply]
大佬
